controls/cf_serverd.cf

```diff
@@ -136,18 +136,36 @@ bundle server access_rules()
       if => isdir( "$(sys.workdir)/ppkeys/" ),
       admit => { @(def.policy_servers) };
 
-      # Allow slave hub to synchronize cf_robot and appsettings content.
+      # Allow slave hub to synchronize cf_robot and appsettings, application
+      # config config, ldap config settings, and ldap api config settings.
```

Comment:
Double *config* here. And also *promsie* in the commit message (twice).

Reply:
application config config was intentional.

application/config/config.php